sdk/search/azure-search-documents/azure/search/documents/_utils.py

```diff
@@ -3,15 +3,31 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-from typing import Any, Optional
+from typing import Any, Optional, Union
 from azure.core.pipeline.policies import (
     BearerTokenCredentialPolicy,
     AsyncBearerTokenCredentialPolicy,
 )
+from ._generated.models import QueryAnswerType
 
 DEFAULT_AUDIENCE = "https://search.azure.com"
 
 
+def get_answer_query(
+    query_answer: Optional[Union[str, QueryAnswerType]] = None,
+    query_answer_count: Optional[int] = None,
+    query_answer_threshold: Optional[float] = None,
+) -> Optional[Union[str, QueryAnswerType]]:
+    if query_answer_count and query_answer_threshold:
+        answers: Optional[Union[str, QueryAnswerType]] = "{}|count-{},threshold-{}".format(
+            query_answer, query_answer_count, query_answer_threshold
+        )
+    else:
+        answers = query_answer if not query_answer_count else "{}|count-{}".format(query_answer, query_answer_count)
+        answers = answers if not query_answer_threshold else "{}|threshold-{}".format(answers, query_answer_threshold)
+    return answers
```

Comment:
Code suggestion that I think would make this easier to parse :)

```suggestion
    answers = query_answer
    separator = "|"
    if query_answer_count:
        answers = f"{answers}{separator}count-{query_answer_count}"
        separator = ","
    if query_answer_threshold:
        answers = f"{answers}{separator}threshold-{query_answer_threshold}"
    return answers
```